header/io/io.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <string>
+
+namespace io
+{
+	std::string fileToString(const std::string& filePath);
+}
+
```

Comment:
Could rename file "io" to "fileToString", as it doesn't seem to be very implicit. Do we have to create a namespace ?